firmware/quintuna/CRIT/src/app/app_mainState.c

```diff
@@ -0,0 +1,28 @@
+#include "app_mainState.h"
```

Comment:
honestly since we have jobs now you don't need a mainstate. at least on this board idk if there are faults, but on other boards fault handling and mainstate kinda get confusing, so it's best to set a good example